semtypes/src/main/java/org/ballerina/semtypes/SemType.java

```diff
@@ -0,0 +1,45 @@
+package org.ballerina.semtypes;
```

Comment:
Lets match the file names, and check if upper case `T` in file name is correct .